src/widgets/current-refinements/__tests__/__snapshots__/current-refinements-test.ts.snap

```diff
@@ -22,16 +22,12 @@ exports[`currentRefinements() render() DOM output renders correctly 1`] = `
         "refinements": Array [
           Object {
             "attribute": "facet",
-            "count": 1,
-            "exhaustive": true,
```

Comment:
what caused these changes?

Reply:
See https://github.com/algolia/instantsearch.js/pull/4000#discussion_r309115051. I think that the `SearchResults` constructor is misused in the tests but I'm not sure how.